### ISSUE TITLE
Kernel module for evdi not compiling

### ISSUE THREAD

**Issue by istathar** [negativo17/evdi-kmod#1]
Trying to get a DisplayLink device going. I installed **akmod-evdi**, **displaylink**, and **libevdi** RPMs from https://negativo17.org/repos/multimedia/fedora-37/x86_64/. It worked!

Unfortunately after a kernel update the external monitor stopped working again.

It's a bit of a strange error. The **akmod-evdi** package installs fine, of course, but when it goes to do its thing it errors out.

Running _akmods_ as root:

```
# akmods
Checking kmods exist for 6.0.16-300.fc37.x86_64            [  OK  ]
Building and installing evdi-kmod                          [FAILED]
Building rpms failed; see /var/cache/akmods/evdi/1.12.0-2.20221013gitbdc258b-for-6.0.16-300.fc37.x86_64.failed.log for details
```

inspecting the log file found this error:

```
2023/01/07 13:30:20 akmodsbuild: cc1: error: code model kernel does not support PIC mode
```

If you want more context I'll post it, but otherwise I'll refrain from spamming this issue with endless lines of cc output.

**Comment by istathar** [comment]
Additional information: it appears the reason it was working before is that I had manually installed the _evdi.ko_ module after building it from source from https://github.com/DisplayLink/evdi; doing that again got the _DisplayLink_ program working again and lo the external monitor is running (and now _akmods_ thinks everything is fine, because there's a module sitting there in _/lib/modules_).

So either I have to accept that the **akmods-evdi** package isn't going to work after all, or figure out how to fix it. I'd be grateful for any tips for how to push on from here.

**Comment by istathar** [comment]
I tried installing your **dkms-evdi** package, and it appears the module got built after and upgrade & reboot (and then akmods helpfully noted that it was already there).

**Comment by scaronni** [comment]
Updated both `evdi-kmod` and `dkms-evdi` to the latest snapshot plus some extra patch to fix recent RHEL (and clones) 8.7/91 kernel builds.

Tested on AlmaLinux 8, 9 and Fedora 37.
